src/components/ResultsList/ResultsList.stories.tsx

```diff
@@ -0,0 +1,49 @@
+// @ts-nocheck
```

Comment:
Thoughts on renaming this to `ResultList` (removing the -s, to be aligned with `ResultItem` (or vice-versa - renaming `ResultItem` to `ResultsItem`)?

Reply:
I think maybe we could (in future) structure these two components in the same folder anyway...

Reply:
The intention is to combine in a future PR. I'll go with ResultsList and ResultsItem since we talk about "show the **results**"